pkg/GraphAppSkeleton/GraphApp.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+import {logFactory, LocalStoragePersistor} from './arcs/arcs.js';
+import {nodeTypes} from './arcs/nodeTypes.js';
+import {services} from './arcs/services.js';
+import {GraphApp as App} from './Library/GraphApp.js';
+import {graph} from './graph.js';
+
+const log = logFactory(true, 'GraphApp', 'navy');
+
+export const GraphApp = class extends App {
```

Comment:
that's confusing having `GraphApp` and `Library/GraphApp`, both having: 
`export const GraphApp = class extends App {`
... but the implementations are different

Reply:
Ah yes, sorry. It's because I've literally factored GraphApp into two pieces, and I want to give us an iteration on that before we promote the *inner* GraphApp to `Library/App` (and fix naming issues at that time).

Reply:
Notice that GraphApp does
import {GraphApp as App} from 'Library/GraphApp.js'
to maximize your confusion.